modules/java_api/build.gradle

```diff
@@ -1,22 +1,25 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        //Check for the latest version here: http://plugins.gradle.org/plugin/com.jfrog.artifactory
-        classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4+"
-    }
-}
-
 plugins {
     id 'java-library'
-    id 'maven-publish'
 }
 
-allprojects {
-    apply plugin: "com.jfrog.artifactory"
+import org.gradle.internal.os.OperatingSystem
+println 'Operating system: ' + OperatingSystem.current()
+
+def nativesCPP;
+def openvinoVersion = "2022.1"
+
+if (OperatingSystem.current().isMacOsX()) {
+    nativesCPP = 'macosx-x86_64'
+} else if (OperatingSystem.current().isLinux()) {
+    nativesCPP = 'linux-x86_64'
+} else if (OperatingSystem.current().isWindows()) {
+    nativesCPP = 'windows-x86_64'
+} else {
+    logger.warn('Unknown operating system!')
 }
 
+project.version = "${openvinoVersion}-${nativesCPP}"
```

Comment:
@likholat, will ot be simpler for your experiment with the samples if I remove a new jar naming? So you can finish the samples building using Gradle and only after that I will check OS specific naming (ot's not critical for this PR).

Reply:
It's not a problem for me)